packages/buidler-core/sample-project/scripts/sample-script.js

```diff
@@ -1,22 +1,13 @@
-// We require the Buidler Runtime Environment explicitly here. This is optional
-// when running the script with `buidler run <script>`: you'll find the Buidler
-// Runtime Environment's members available as global variable in that case.
-const env = require("@nomiclabs/buidler");
-
 async function main() {
-  // You can run Buidler tasks from a script.
-  // For example, we make sure everything is compiled by running "compile"
-  await env.run("compile");
+  // We get the contract to deploy
+  const Greeter = await ethers.getContractFactory("Greeter");
+  const greeter = await Greeter.deploy("Hello, Buidler!");
 
-  // We require the artifacts once our contracts are compiled
-  const Greeter = env.artifacts.require("Greeter");
-  const greeter = await Greeter.new("Hello, world!");
+  await greeter.deployed();
 
-  console.log("Greeter address:", greeter.address);
+  console.log("Greeter deployed to:", greeter.address);
 }
 
-// We recommend this pattern to be able to use async/await everywhere
-// and properly handle errors.
```

Comment:
why remove this?

Reply:
I asked @alcuadrado to remove this in order to simplify the script. There are two main reasons:

1. Extending the BRE is an advanced use and is explained in `/advanced/buidler-runtime-environment.html`
2. When running `npx buidler run scripts/sample-script.js` the contracts were compiling twice.

Reply:
My bad. I copied the script from the tutorial, should have checked the comments before commiting.

Re-adding those lines.